appserver/web/weld-integration/src/main/java/org/glassfish/cdi/transaction/TransactionScopedCDIUtil.java

```diff
@@ -219,7 +219,7 @@ public BeanWrapper(Class<?> beanClass) {
             catch(MultiException e) {
                 log(e.getMessage());
             }
-            this.ctxUtil = ctxUtil;
+            this.ctxUtil = ctxUtil.map(JavaEEContextUtil::currentInvocation);
```

Comment:
Context Util API changed, this line just tracks that change, it would be a compile error otherwise.
It's self-explanatory for me here and shouldn't require a comment, as it would be like
"tracking API changes"